app/model/db/idx_token.py

```diff
@@ -0,0 +1,321 @@
+"""
+Copyright BOOSTRY Co., Ltd.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+
+See the License for the specific language governing permissions and
+limitations under the License.
+
+SPDX-License-Identifier: Apache-2.0
+"""
+from typing import Union, Type
+from sqlalchemy import (
+    Column,
+    String,
+    BigInteger,
+    Boolean,
+    Float,
+    Text,
+    JSON,
+)
+from app.model.db import Base
+
+
+class TokenBase(Base):
+    """Token Base Attribute"""
+    __abstract__ = True
```

Comment:
Added all attributes of tokens and avoid holding them as JSON schema.

Pros:
- we could use them as search key
Cons:
- some difficulty of migration(JSON schema may also have)